js-miniapp-sample/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import "./App.css";
```

Comment:
I think this CSS file can be removed since it's not used (there are no HTML elements with an `.App` class)

Reply:
https://github.com/rakutentech/js-miniapp/blob/34389703dae56344d2824442053dc4f9b19c832e/js-miniapp-sample/src/App.js#L20

updated it to use makeStyles css style